src/components/product/product.service.ts

```diff
@@ -406,30 +370,26 @@ export class ProductService {
       };
     }
 
-    const scriptureReferencesOverride = await this.listScriptureReferences(
-      baseNodeProps.id,
+    const scriptureReferencesOverrideValue = await this.listScriptureReferences(
+      id,
       'Product',
       session,
       { isOverride: true }
     );
 
-    const typeName = difference(produces.p.labels, [
-      'Producible',
-      'BaseNode',
-    ])[0];
+    const typeName = difference(pr.p.labels, ['Producible', 'BaseNode'])[0];
 
     const producible = await this.getProducibleByType(
-      produces.p.properties.id,
+      pr.p.properties.id,
       typeName,
       session
     );
 
     return {
-      ...baseNodeProps,
+      ...parseBaseNodeProperties(result.node),
       ...securedProperties,
       scriptureReferences: {
-        canRead: result.canScriptureReferencesRead,
-        canEdit: result.canScriptureReferencesEdit,
+        ...securedProperties.scriptureReferences,
         value: [],
```

Comment:
This should be the `scriptureReferencesOverride` if it's not null or the `produces.scriptureReferences`

Reply:
Ahh, got it.